api/peripheral_api/models/PrintLog.js

```diff
@@ -9,7 +9,6 @@ const PrintLogSchema = new Schema(
     },
     chosenPrinter: {
       type: String,
-      required: true
```

Comment:
removing this for now because core-v4 has no way of knowing which printer was chosen atm